theme-changer.el

```diff
@@ -130,8 +136,10 @@ Returns the theme that was enabled."
         (enable (if (not (string= theme-changer-mode "deftheme"))
                     (lambda () (apply (symbol-function new) '()))
                   (lambda () (load-theme new t)))))
+    (run-hook-with-args theme-changer-pre-change-functions old)
     (disable-theme old)
     (if new (funcall enable))
+    (run-hook-with-args theme-changer-post-change-functions new)
```

Comment:
Hi @hadronzoo
First of all I appreciate create & publish this package.

I think you need to insert the single quote before `theme-changer-post-change-functions`.
I mean above line should be like this:
```elisp
(run-hook-with-args 'theme-changer-post-change-functions new)
```